sdk/storage/azure-storage-file-datalake/tests/test_file_system.py

```diff
@@ -75,6 +77,162 @@ def test_create_file_system(self, datalake_storage_account_name, datalake_storag
         # Assert
         self.assertTrue(created)
 
+    @DataLakePreparer()
+    def test_create_file_owner_with_file_system(self, datalake_storage_account_name, datalake_storage_account_key):
```

Comment:
As with a previous comment, we can probably forgo the file_system tests all together? `test_file` should cover everything here. All the file_system APIs do is create a FileClient and then call that method, so if that method is tested, I think that's enough.